gamestonk_terminal/portfolio/portfolio_optimization/optimizer_view.py

```diff
@@ -160,7 +160,7 @@ def display_max_sharpe(
     else:
         print("\n", s_title)
         display_weights(weights)
-    ef.portfolio_performance(verbose=True)
+    ef.portfolio_performance(verbose=True, risk_free_rate=get_rf())
```

Comment:
still want the user to be able to define a rfrate

Reply:
I was thinking about that myself. Where could we maintain the custom -r? I don't think it happened before my pr either.

Reply:
I think the best solution would be similar to what you have implemented.  Have get_rf() be the default rfrate in the parser, so if it not supplied then we use the T Bill.  Then only use the rfrate in your functions as was, not redefining them in _view.

Reply:
I see!!! Thank you changes made.